utilities/sinsp-builder.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# This script is meant to be run from inside a collector-builder image. It will
+# create the driver for the current system, as well as Falco's sinsp-example
+# binary, which is useful for quickly testing driver changes.
+#
+# Run the collector-builder image with the following command:
+#   docker run --rm -it --entrypoint bash \
+#       --privileged \
+#       -v /var/run/docker.sock:/var/run/docker.sock \
+#       -v /sys:/sys:ro \
+#       -v /dev:/dev \
+#       -v /tmp:/tmp \
+#       quay.io/stackrox-io/collector-builder:cache
+#
+# If you are running on an immutable system, create a docker volume then add
+# the following arguments:
+#   -v <your-volume>:/tmp/collector -e DEV_SHARED_VOLUME=<your-volume>
+#
+# If you are lazy like me, you don't need to clone the collector repo before
+# executing, you can simply run this inside the container:
+#   curl https://raw.githubusercontent.com/stackrox/collector/master/utilities/sinsp-builder.sh | sh
```

Comment:
If this is run before this PR is merged the correct command is `curl https://raw.githubusercontent.com/stackrox/collector/mauro-easy-debug-drivers/utilities/sinsp-builder.sh | sh`

Reply:
This did build the sinsp-example binary, but it also exited with a failure 

```[ 97%] Building CXX object libsinsp/examples/CMakeFiles/sinsp-example.dir/test.cpp.o
[ 98%] Building CXX object libsinsp/examples/CMakeFiles/sinsp-example.dir/util.cpp.o
[100%] Linking CXX executable sinsp-example
make[3]: Leaving directory '/tmp/tmp.8hp0tSs4Yb'
[100%] Built target sinsp-example
make[2]: Leaving directory '/tmp/tmp.8hp0tSs4Yb'
make[1]: Leaving directory '/tmp/tmp.8hp0tSs4Yb'
make: Leaving directory '/tmp/tmp.8hp0tSs4Yb'
gzip: /tmp/collector/kernel-modules/container/kernel-modules/*.gz: No such file or directory
[root@999f0c8093a0 install-tmp]# echo $?
1
```

Reply:
It errored out when trying to decompress the kernel drivers, could you double check if those were properly built?

Reply:
I had this problem - it's because the images used do not contain the changes in this PR (addition of cmake, and adjustment to the `drivers` make target) This won't be a problem once this PR is merged, and the workaround for testing now is to rebuild the images locally with the changes